migrations/20171026184208_create_post_data_export_job_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class CreatePostDataExportJobTable extends AbstractMigration
+{
+
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('postdataexport_job')
```

Comment:
we might want to have the author info here actually

Reply:
@willdoran after the discussion with Jess and Justin we only need user id, not author, since anon won't be able to export.